reports/report.json

```diff
@@ -1197,92 +1379,110 @@
+            "src_char": "2103:18"
           },
           {
             "contract_path": "src/ConstantsLiterals.sol",
             "line_no": 25,
-            "src": "958:3"
+            "src": "958:3",
+            "src_char": "958:3"
           },
           {
             "contract_path": "src/ConstantsLiterals.sol",
             "line_no": 26,
-            "src": "992:3"
+            "src": "992:3",
+            "src_char": "992:3"
           },
           {
             "contract_path": "src/ConstantsLiterals.sol",
             "line_no": 27,
-            "src": "1032:4"
+            "src": "1032:4",
+            "src_char": "1032:4"
           },
           {
             "contract_path": "src/ConstantsLiterals.sol",
             "line_no": 28,
-            "src": "1068:4"
+            "src": "1068:4",
+            "src_char": "1068:4"
           },
           {
             "contract_path": "src/ConstantsLiterals.sol",
             "line_no": 29,
-            "src": "1117:42"
+            "src": "1117:42",
+            "src_char": "1117:42"
           },
           {
             "contract_path": "src/ConstantsLiterals.sol",
             "line_no": 30,
-            "src": "1192:42"
+            "src": "1192:42",
+            "src_char": "1192:42"
           },
           {
             "contract_path": "src/ConstantsLiterals.sol",
             "line_no": 31,
-            "src": "1275:66"
+            "src": "1275:66",
+            "src_char": "1275:66"
           },
           {
             "contract_path": "src/ConstantsLiterals.sol",
             "line_no": 32,
-            "src": "1374:66"
+            "src": "1374:66",
+            "src_char": "1374:66"
           },
           {
             "contract_path": "src/RevertsAndRequriesInLoops.sol",
             "line_no": 10,
-            "src": "253:2"
+            "src": "253:2",
+            "src_char": "253:2"
           },
           {
             "contract_path": "src/RevertsAndRequriesInLoops.sol",
             "line_no": 17,
-            "src": "442:2"
+            "src": "442:2",
+            "src_char": "442:2"
           },
           {
             "contract_path": "src/RevertsAndRequriesInLoops.sol",
             "line_no": 26,
-            "src": "706:2"
+            "src": "706:2",
+            "src_char": "706:2"
           },
           {
             "contract_path": "src/RevertsAndRequriesInLoops.sol",
             "line_no": 27,
-            "src": "745:2"
+            "src": "745:2",
+            "src_char": "745:2"
           },
           {
             "contract_path": "src/eth2/DepositContract.sol",
             "line_no": 113,
-            "src": "7252:2"
+            "src": "7252:2",
+            "src_char": "5764:2"
           },
           {
             "contract_path": "src/eth2/DepositContract.sol",
             "line_no": 113,
-            "src": "7293:2"
+            "src": "7293:2",
+            "src_char": "5805:2"
           }
```

Comment:
`src` => `byte_offset : byte_len`
`src_char` => `char_offset : char_len`

Difference is visible in files with non ascii characters :)